hoot-core/src/main/cpp/hoot/core/conflate/tile/TileUtils.cpp

```diff
@@ -109,70 +110,31 @@ geos::geom::Envelope TileUtils::getRandomTile(
   return geos::geom::Envelope();
 }
 
-void TileUtils::writeTilesToGeoJson(
-  const std::vector<std::vector<geos::geom::Envelope>>& tiles,
+void TileUtils::writeTilesToGeoJson(const std::vector<std::vector<geos::geom::Envelope>>& tiles,
   const std::vector<std::vector<long>>& nodeCounts, const QString& outputPath,
-  const bool selectSingleRandomTile, int randomSeed)
+  const QString& fileSource, const bool selectSingleRandomTile, int randomSeed)
 {
-  // write out to temp osm and then use ogr2ogr to convert to geojson; trying to remember, but I
-  // believe our geojson writer wouldn't quite give the geometry output format needed for this,
-  // which is why ogr2ogr is used here...but that's worth verifying now.
```

Comment:
With the changes to the `OsmGeoJson` this is no longer true.  The changes almost exactly replicate the `ogr2ogr` output without shelling out of Hootenanny.  That makes most of the code below obsolete too.